### ISSUE TITLE
Declare 4.30 RC2

### ISSUE THREAD

**Issue by MohananRahul** [eclipse-platform/eclipse.platform.releng.aggregator#1564]
Declare 4.30 RC2
A "go" is needed from all the components below to declare this milestone.

Eclipse downloads:
https://download.eclipse.org/eclipse/downloads/drops4/I20231122-1800

Build logs and/or test results (eventually):
https://download.eclipse.org/eclipse/downloads/drops4/I20231122-1800/testResults.php

Software site repository:
https://download.eclipse.org/eclipse/updates/4.30-I-builds

Specific (simple) site repository:
https://download.eclipse.org/eclipse/updates/4.30-I-builds/I20231122-1800

Equinox downloads:
https://download.eclipse.org/equinox/drops/I20231122-1800

----

Deadlines:

Friday, 2023-November-24, around 1 AM (Eastern): deadline for sign-off (or, by then, comment in this issue when you expect to sign-off).
Friday, 2023-November-24, around 3 AM (Eastern): promote approved build to S-4.30RC2-*, contribute to simultaneous release repo, and announce to mailing lists.
Remember to investigate and document here any failing JUnit tests.

----

- [ ] Platform: 
  - [ ]  Resources 
  - [ ]  UI 
  - [ ]  Debug 
  - [ ]  Ant 
  - [ ]  SWT 
  - [ ]  Releng
- [ ]  JDT:
  - [ ]  Core 
  - [ ]  Debug 
  - [ ]  UI
- [ ]  PDE
- [ ]  Equinox

**Comment by HannesWell** [comment]
Go from PDE.

**Comment by jarthana** [comment]
JDT would like to make [some contribution to the N&N](https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/93). Can we have another build please? The changes are already in the JDT news page but yet to be included in the releng repo.

**Comment by MohananRahul** [comment]
@alshamams is working on it once done ,will trigger another build.

**Comment by merks** [comment]
I didn't know the N&N were built-in!  😱   I wonder if we should also incorporate the following in that case?

https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/95

**Comment by jarthana** [comment]
> I didn't know the N&N were built-in! 😱 I wonder if we should also incorporate the following in that case?
> 
> [eclipse-platform/www.eclipse.org-eclipse#95](https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/95)

I am fine with it. But not sure if @tjwatson will have a chance to respond by then.

**Comment by merks** [comment]
Yes, he won't have time, but I assume he'll be find with this...

**Comment by iloveeclipse** [comment]
Please also note broken links, see most resent help test results .

**Comment by MohananRahul** [comment]
> Please also note broken links, see most resent help test results .

@iloveeclipse you are referencing to https://download.eclipse.org/eclipse/downloads/drops4/I20231122-1800/testresults/html/org.eclipse.ua.tests.doc_ep430I-unit-mac64-java17_macosx.cocoa.x86_64_17.html result right?

**Comment by iloveeclipse** [comment]
Exact

**Comment by tjwatson** [comment]
> > I didn't know the N&N were built-in! 😱 I wonder if we should also incorporate the following in that case?
> > 
> > [eclipse-platform/www.eclipse.org-eclipse#95](https://github.com/eclipse-platform/www.eclipse.org-eclipse/pull/95)
> 
> I am fine with it. But not sure if @tjwatson will have a chance to respond by then.

Fine with me.

**Comment by MohananRahul** [comment]
https://ci.eclipse.org/releng/job/Builds/job/I-build-4.30/140/ Started new build

**Comment by iloveeclipse** [comment]
https://github.com/eclipse-platform/eclipse.platform/pull/869 is most likely needed, but it has API errors because of the bug/feature in tycho.

**Comment by iloveeclipse** [comment]
> [eclipse-platform/eclipse.platform#869](https://github.com/eclipse-platform/eclipse.platform/pull/869) is most likely needed, but it has API errors because of the bug/feature in tycho.

API errors were resolved, https://github.com/eclipse-platform/eclipse.platform/pull/869 merged.

New build triggered: https://ci.eclipse.org/releng/job/Builds/job/I-build-4.30/141/

**Comment by iloveeclipse** [comment]
Hopefully last RC2 build candidate : https://download.eclipse.org/eclipse/downloads/drops4/I20231123-1700/

**Comment by noopur2507** [comment]
Go from JDT UI.

**Comment by jarthana** [comment]
Go from JDT Core

**Comment by subyssurendran666** [comment]
Go from JDT.Debug

**Comment by subyssurendran666** [comment]
Go from Platform.Debug

**Comment by deepika-u** [comment]
Go from SWT

**Comment by MohananRahul** [comment]
Go from Releng

**Comment by MohananRahul** [comment]
No change ECJ. 
Contributed and announced.
